### PR TITLE
app-containers/podman-tui: add 0.11.0 and live

### PR DESCRIPTION
**Pull request by rahilarious**
Terminal UI frontend for Podman

### CODE REVIEW

**Review by zmedico — APPROVED** [review]
Looks good!

File: app-containers/podman-tui/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
```
Comment (thesamesam) [inline app-containers/podman-tui/metadata.xml:8]: The proxy-maint project should be last. You want bugs assigned to you.

File: app-containers/podman-tui/podman-tui-0.11.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+DESCRIPTION="Terminal UI frontend for Podman"
+HOMEPAGE="https://github.com/containers/podman-tui"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/podman-tui.git"
+else
+	SRC_URI="https://github.com/containers/podman-tui/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+LICENSE="Apache-2.0 BSD-2 BSD MIT MPL-2.0"
```
Comment (thesamesam) [inline app-containers/podman-tui/podman-tui-0.11.0.ebuild:16]: newline before LICENSE

File: app-containers/podman-tui/podman-tui-0.11.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+DESCRIPTION="Terminal UI frontend for Podman"
+HOMEPAGE="https://github.com/containers/podman-tui"
```
Comment (thesamesam) [inline app-containers/podman-tui/podman-tui-0.11.0.ebuild:8]: newline after HOMEPAGE

**Comment by rahilarious** [comment]
@zmedico this seems ready to merge

**Pull request closed without merging**